src/pybind/mgr/influx/module.py

```diff
@@ -264,7 +307,7 @@ def handle_command(self, cmd):
         if cmd['prefix'] == 'influx self-test':
             daemon_stats = self.get_daemon_stats()
             assert len(daemon_stats)
-            df_stats = self.get_df_stats()
+            df_stats = self.get_df_stats()[0]
```

Comment:
Should also call `get_pg_summary` during selftest in order to cover it in testing